charts/delphai-deployment/values.yaml

```diff
@@ -135,6 +140,17 @@ ingress:
   # Run Cloudflare purge_cache after deployment
   purgeCloudflareCache: false
 
+rabbitmq:
```

Comment:
Shouldn't we call this section `autoscaling` and make it less rabbit-specific? We might eventually want to configure standard HPA or other keda triggers. Since we already have min-max replicas here

Reply:
Renamed in c6e16946b078266b443c3fc4357e8212bbb3406a